aptos-move/framework/move-stdlib/sources/vector.move

```diff
@@ -589,6 +592,21 @@ module std::vector {
         for_each_reverse(v, |e| d(e))
     }
 
+    public fun range(start: u64, end: u64): vector<u64> {
```

Comment:
You need a mut reference to the vector for both the functions

Reply:
Sounds good, will add.